app/assets/stylesheets/era_custom.scss

```diff
@@ -16,3 +22,46 @@ div.terms-of-use-custom {
   margin-left: 30px;
   margin-right: 30px;
 }
+
+.dark {
+  background-color: #2d353d;
+  color: #fff;
+}
+
+.footer {
```

Comment:
This is fine, but I am wondering if a lot of this custom css is needed? 

For example this footer is in a row-fluid class (which should give the width 100% anyways), can throw a `.text-center` class on it instead of adding it here. 

Font size? Do we really need to play around with this? H2 will be bigger by default, everything else should be fine how it is. (can throw copyright in a `<small>` tag if you want it smaller, etc)

Same with  alot of the unorderlist custom css you have like `.footer ul`. Bootstrap has a style for this already for you! Simply just add a class of `list-unstyled` on the `<ul>` tag.

Reply:
Footer CCS (and most code) pulled directly from UAL Avalon 5. I did not take the time to rationalize to help in finishing closer to the deadline. 

I agree with @murny  point in the review.